app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,18 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    
+    
+    
+    
+    <uses-permission android:name="android.permission.MANAGE_USERS" />
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: ProtectedPermissions
**Severity**: ERROR
**File**: C:/Users/anisart/.jenkins/workspace/test3/app/src/main/AndroidManifest.xml L17

Using system app permission&#10;Permission is only granted to system apps&#10;Permissions with the protection level signature, privileged or signatureOrSystem are only granted to system apps. If an app is a regular non-system app, it will never be able to use these permissions.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1563095342>*